core/src/Log.purs

```diff
@@ -183,25 +182,18 @@ justOrDieWith' value msg = case value of
     die' msg
 
 rightOrDie :: ∀ b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable err => Either err x -> m x
-rightOrDie value = rightOrDieWith value identity
+rightOrDie = rightOrDieWith identity
 
 rightOrDie_ :: ∀ b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable err => Either err x -> m Unit
 rightOrDie_ = void <<< rightOrDie
 
-rightOrDieWith :: ∀ a b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable a => Either err x -> (err -> a) -> m x
-rightOrDieWith value toMsg = case value of
+rightOrDieWith :: ∀ a b m err x. MonadEffect m => MonadAsk (LogEnv b) m => Loggable a => (err -> a) -> Either err x -> m x
+rightOrDieWith toMsg value = case value of
```

Comment:
This function wasn't actually used anywhere (until now), so I swapped the parameters to allow it to be called with postfix application like `foo >>= rightOrDieWith \err -> die $ "Error: " <> err`